apps/dav/lib/Connector/Sabre/QuotaPlugin.php

```diff
@@ -88,6 +91,31 @@ public function beforeCreateFile($uri, $data, INode $parent, $modified) {
 		return $this->checkQuota($parent->getPath() . '/' . basename($uri));
 	}
 
+	/**
+	 * Check quota before creating directory
+	 *
+	 * @param RequestInterface $request
+	 * @param ResponseInterface $response
+	 * @return bool
+	 * @throws InsufficientStorage
+	 * @throws \Sabre\DAV\Exception\Forbidden
+	 */
+	public function onCreateCollection(RequestInterface $request, ResponseInterface $response): bool {
+		try {
+			$destinationPath = $this->server->calculateUri($request->getUrl());
+			$quotaPath = $this->getPathForDestination($destinationPath);
+		} catch (\Exception $e) {
+			return true;
+		}
+		if ($quotaPath) {
+			// MKCOL does not have a Content-Length header, so we can use
+			// a fixed value for the quota check.
+			return $this->checkQuota($quotaPath, 4096);
```

Comment:
```suggestion
			return $this->checkQuota($quotaPath, 4096, true);
```

I suppose the new option was supposed to be used?

Reply:
Ah, yes, good eye!